dist/maven-antrun/build-parallel-worlds.xml

```diff
@@ -131,7 +131,7 @@
 
         <!-- Remove the explicily unshimmed files from the common directory -->
         <delete>
-            <fileset dir="${project.build.directory}/parallel-world/spark3xx-common"
+            <fileset dir="${project.build.directory}/parallel-world/spark34-common"
```

Comment:
spark34 may confusingly suggest spark 3.4 for the reader

Let us just do spark-common or spark-shared. spark3xx was not an ideal choice. What if this code survives until Spark 5 :)